src/main/java/com/github/princesslana/slothbot/MessageCounter.java

```diff
@@ -81,6 +106,53 @@ public void clearBuckets(String channelId) {
                 .collect(ImmutableList.toImmutableList()));
   }
 
+  public boolean getBotCounterConfig(String guildId) {
+    return Optional.ofNullable(botCounterConfigs.get(guildId)).orElse(false);
```

Comment:
`getOrDefault` could be used instead of the Optional stuff here

Reply:
I didn't know that was a method, I'll change to that.